src/triangleNode.js

```diff
@@ -0,0 +1,18 @@
+(function(global) {
+  class triangleNode extends NIN.ShaderNode {
+    constructor(id, options) {
+      console.log("options for triangle", options);
+      options.shader = SHADERS.triangle;
+
+      this.quad.material.uniforms.
```

Comment:
Loads of "lols" nowadays. Probably because Lejla is watching Paradise Hotel in the background.